.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I guess we need to keep the lock file.

Reply:
As a lib, it should always be noticed immediately if it can no longer be built due to dependencies. So I recently found out that "compose" is missing as a method. I will keep these, otherwise I'll be in the dark and only notice these effects later in the project, when it goes into the productive system.

Of course, I can also understand the advantages.